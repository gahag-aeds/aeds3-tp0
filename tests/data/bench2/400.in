400 200
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 
Add 95 227
Max 137 269
Sub 139 271
Sum 105 237
Add 22 154
Max 89 221
Sub 28 160
Sum 42 174
Sub 129 261
Max 122 254
Sub 14 146
Min 202 334
Sub 49 181
Max 119 251
Sub 242 374
Sum 132 264
Sub 252 384
Min 91 223
Add 245 377
Sum 243 375
Add 151 283
Max 101 233
Sub 2 134
Max 254 386
Add 80 212
Min 256 388
Add 181 313
Max 92 224
Add 152 284
Max 41 173
Add 48 180
Min 1 133
Add 177 309
Max 209 341
Add 105 237
Min 220 352
Add 30 162
Min 154 286
Sub 16 148
Sum 180 312
Sub 30 162
Sum 30 162
Add 114 246
Sum 78 210
Add 170 302
Max 89 221
Add 231 363
Min 154 286
Sub 201 333
Min 208 340
Add 151 283
Sum 84 216
Add 63 195
Max 152 284
Sub 93 225
Max 164 296
Add 140 272
Sum 6 138
Add 10 142
Max 47 179
Sub 68 200
Min 58 190
Sub 47 179
Max 266 398
Sub 266 398
Sum 174 306
Sub 218 350
Max 50 182
Sub 60 192
Max 233 365
Sub 252 384
Sum 90 222
Sub 17 149
Max 98 230
Sub 189 321
Sum 186 318
Sub 208 340
Sum 153 285
Add 72 204
Sum 141 273
Add 115 247
Min 223 355
Add 247 379
Max 200 332
Add 129 261
Sum 72 204
Sub 118 250
Max 2 134
Add 99 231
Max 128 260
Add 70 202
Min 166 298
Sub 185 317
Sum 138 270
Add 164 296
Min 205 337
Sub 44 176
Min 136 268
Sub 9 141
Max 104 236
Sub 123 255
Max 260 392
Add 215 347
Max 140 272
Add 91 223
Min 136 268
Add 58 190
Max 53 185
Sub 43 175
Max 152 284
Add 216 348
Sum 180 312
Add 129 261
Sum 195 327
Sub 134 266
Max 257 389
Sub 21 153
Max 251 383
Sub 13 145
Max 119 251
Add 111 243
Max 104 236
Add 17 149
Max 50 182
Sub 242 374
Min 202 334
Sub 254 386
Sum 18 150
Sub 70 202
Max 263 395
Sub 143 275
Sum 192 324
Sub 255 387
Sum 90 222
Sub 86 218
Min 100 232
Sub 248 380
Min 166 298
Sub 175 307
Sum 45 177
Sub 50 182
Sum 123 255
Add 246 378
Max 200 332
Sub 72 204
Min 112 244
Sub 211 343
Max 92 224
Add 96 228
Sum 246 378
Sub 232 364
Sum 228 360
Add 82 214
Max 248 380
Add 11 143
Max 56 188
Sub 204 336
Sum 264 396
Add 74 206
Sum 6 138
Add 14 146
Sum 216 348
Add 219 351
Sum 24 156
Sub 61 193
Max 38 170
Sub 145 277
Sum 63 195
Add 203 335
Min 52 184
Add 107 239
Min 7 139
Sub 196 328
Max 86 218
Add 228 360
Sum 267 399
Add 219 351
Min 193 325
Add 113 245
Max 47 179
Add 171 303
Max 77 209
Sub 30 162
Min 253 385
Add 79 211
Min 40 172
Add 41 173
Sum 15 147
Add 58 190
Min 136 268
Sub 42 174
Sum 72 204
Add 107 239
Sum 261 393
Sub 117 249
Max 167 299
Add 53 185
Sum 261 393
Sub 229 361
Sum 255 387
