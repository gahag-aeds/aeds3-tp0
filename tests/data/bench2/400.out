100
6575
99
6454
98
-1
97
6777
-2
6341
98
98
-1
99
100
-2
100
-1
2
7327
6068
6959
103
4
-1
7096
105
104
6216
105
-1
98
7511
103
102
7010
103
7226
7310
7493
-5
101
6928
96
105
-1
7651
-4
2
106
97
105
3
107
106
7596
7267
97
101
106
107
107
-3
5782
97
7191
7158
3
-3
6043
7293
102
1
102
5980
6129
97
104
5656
5888
6120
5974
104
6554
-6
-6
105
5955
-5
105
107
-5
-7
6006
-2
7103
5939
108
5939
5936
